tests/quantization/torchao_integration/test_torchao.py

```diff
@@ -451,12 +446,11 @@ def check_serialization_expected_output(self, device, expected_output):
         """
         Test if we can serialize and load/infer the model again on the same device
         """
-        dtype = torch.bfloat16 if self.quant_scheme == "int4_weight_only" else "auto"
+        dtype = torch.bfloat16
         with tempfile.TemporaryDirectory() as tmpdirname:
-            self.quantized_model.save_pretrained(tmpdirname, safe_serialization=False)
+            self.quantized_model.save_pretrained(tmpdirname, safe_serialization=True)
```

Comment:
we can add a new test for Float8WeightOnlyConfig and Float8DynamicActivationFloat8WeightConfig I think, and revert all the changes to previous tests